Framework/API/src/ExperimentInfo.cpp

```diff
@@ -166,7 +166,19 @@ makeDetectorInfo(const Instrument &oldInstr, const Instrument &newInstr) {
   } else {
     // If there is no DetectorInfo in the instrument we create a default one.
     const auto numDets = oldInstr.getNumberDetectors();
-    return Kernel::make_unique<Beamline::DetectorInfo>(numDets);
+    // Currently monitors flags are stored in the detector cache of the base
```

Comment:
Ok. This is a very small amount of data in any case. But I agree this should eventually live on the `DetectorInfo` rather than the `Instrument`